apps/files_trashbin/lib/Sabre/TrashRoot.php

```diff
@@ -26,6 +27,10 @@ public function __construct(
 	}
 
 	public function delete() {
+		if (!ConfigService::getDeleteFromTrashEnabled()) {
+			throw new Forbidden('Not allowed to delete trash in the trashbin');
```

Comment:
"delete trash in the trashbin" looks odd to me.

```suggestion
			throw new Forbidden('Not allowed to items from the trashbin');
```